packages/react/src/patterns/blocks/ContentBlockMixedGroups/ContentBlockMixedGroups.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { ContentBlock } from '../../sub-patterns/ContentBlock';
+import { ContentGroupCards } from '../ContentGroupCards';
+import { ContentGroupPictograms } from '../ContentGroupPictograms';
+import { ContentGroupSimple } from '../ContentGroupSimple';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * ContentBlockMixedGroups Pattern
+ *
+ * @param {object} props ContentBlockMixedGroups props object
+ * @param {string} props.heading Content block heading
+ * @param {string} props.copy Content block description
+ * @param {*} props.children Content block mixed groups
+ * @param {*} props.cta cta object
+ * @returns {*} ContentBlockMixedGroups JSX Object
+ */
+const ContentBlockMixedGroups = ({ heading, copy, children, cta }) => (
+  <ContentBlock
+    heading={heading}
+    copy={copy}
+    cta={cta}
+    customClassName={`${prefix}--content-block-mixedgroups ${prefix}--col-lg-8`}>
+    <div data-autoid={`${stablePrefix}--content-block-mixedgroups`}>
+      {children}
+    </div>
```

Comment:
So one big change here I think is we need to loop through an array of items. Each item should  have a type that we can use to determine which content group to leverage. The rest of the data in that item should be the props for that content group. Reach out if you have any questions.

Reply:
In the specs it's children, It should be item. I missed this in updating it, so sorry about that name change.